docker-compose.yaml

```diff
@@ -10,3 +10,15 @@ services:
     ports:
       - 3335:3335
       - 9229:9229
+  postgres:
+    image: bitnami/postgresql:latest
```

Comment:
Ao invés de colocar latest na tag de versão da imagem, recomendo deixar explicito qual é a versão que está sendo utilizada.

Se deixa dessa maneira toda vez que tu for rodar o docker compose up numa máquina nova ou tiver apagado os containers, é provável que ele puxe a última versão da imagem e não que estamos utilizando, o que poder gerar problemas de compatibilidade.